src/main/java/homework_4/singleton/Main.java

```diff
@@ -0,0 +1,7 @@
+package homework_4.singleton;
+
+public class Main {
+    public static void main(String[] args) {
+
+    }
+}
```

Comment:
Opt: why do you need it?

Reply:
Forgot to delete it after checks... Fixed.